Visiual(s)

GreatAngel01 opened the pull request:
Изменение отрисовки тайлов ,изменение спрайтов
изменение отрисовки поворота персонажа на поворот холста
добавлены спрайты и внедрена генерация Ильи

Осторожнее с боевкой
Она может быть устаревшей

GreatAngel01 committed:
PreItemSystem+DrawMapOnButtonM+

GreatAngel01 committed:
1.1

GreatAngel01 committed:
1.2

GreatAngel01 committed:
Merge remote-tracking branch 'origin/visiual(S)' into visiual(S)

GreatAngel01 committed:
1.2

GreatAngel01 committed:
1.3

GreatAngel01 committed:
Относительные координаты и fullscreen

GreatAngel01 committed:
Старая боевка
fullscrean+генерация ильи

GreatAngel01 committed:
изменение спрайтов

GreatAngel01 committed:
изменение спрайтов

GreatAngel01 committed:
изменение спрайтов+
изменение типа отрисовки на повороты
изменение отрисовки стен

GreatAngel01 committed:
изменение отрисовки стен

masamonoke reviewed (changes requested):
1. Падает при числе тайлов > 70
2. По идее начальные значения в Visual Data, такие как размеры тайла, персонажа и размеры окна должны задаваться из конструктора
3. И где хотя бы минимальные тесты

GreatAngel01 commented:
1) Я хз как править, ошибка та же что и обычно, на это выделенное количество памяти не влияет
2) Ок,можно доделать,но не критично
3) сделаю

masamonoke commented:
Просто переставь на 70 размер карты. Мне вообще больше интересно, чтобы ты сделал новый пр и посмотреть, что будет. По-идее он должен автоматически к текущему подцепиться

GreatAngel01 committed:
изменение отрисовки стен

GreatAngel01 committed:
Изменение отрисовки карты на
центрирование на персонажа

GreatAngel01 committed:
Изменение отрисовки карты на
центрирование на персонажа

GreatAngel01 committed:
Изменение отрисовки карты на
центрирование на персонажа

GreatAngel01 committed:
Изменение отрисовки карты на
центрирование на персонажа

masamonoke committed:
Merge branch 'master' into visiual(S)

# Conflicts:
#	src/main/java/com/vsu/visual/CharacterControls.java
#	src/main/java/com/vsu/visual/VisualData.java
#	src/main/java/com/vsu/visual/drawers/CharacterDrawer.java
#	src/main/java/com/vsu/visual/drawers/GameDrawer.java

masamonoke committed:
добавлены хендлеры под мувменты
переделан Actor под компонентную структуру
добавлены временные эффекты: возможно делать баффы и дебаффы
переделан алгоритм формирования комбо дерева
переделан формат файла для комбо дерева, добавлен отдельный класс парсер
добавлена возможность чтения json из файла
много тестов
